src/win/tcp.c

```diff
@@ -1446,6 +1446,8 @@ int uv_tcp_open(uv_tcp_t* handle, uv_os_sock_t sock) {
   WSAPROTOCOL_INFOW protocol_info;
   int opt_len;
   int err;
+  struct sockaddr_storage saddr;
+  int saddr_len = sizeof(saddr);
```

Comment:
Minor stylistic issue: can you change this to just `int saddr_len;` and do the assignment a few lines below?